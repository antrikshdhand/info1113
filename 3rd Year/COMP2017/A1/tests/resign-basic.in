place N4
place M1
place K8
who
resign
