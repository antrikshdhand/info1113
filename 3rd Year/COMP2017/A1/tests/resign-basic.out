W
Black wins!
N4M1K8
Thank you for playing!
